deployment/cdk/opensearch-service-migration/default-values.json

```diff
@@ -12,7 +12,7 @@
   "migrationAssistanceEnabled": true,
   "replayerOutputEFSRemovalPolicy": "DESTROY",
   "migrationConsoleServiceEnabled": true,
-  "trafficReplayerServiceEnabled": true,
+  "trafficReplayerServiceEnabled": false,
```

Comment:
With this change we need to add it as true in test/defaultMigrationContext.json and in the cdk demo deploy